core/tee/tee_ree_fs.c

```diff
@@ -245,339 +162,79 @@ static int tee_fs_send_cmd(struct tee_fs_rpc *bf_cmd, void *data, uint32_t len,
  */
 #define REE_FS_NAME_MAX (TEE_FS_NAME_MAX + 20)
 
```

Comment:
as all routnies below only call the generic `tee_fs_rpc_xxx(OPTEE_MSG_RPC_CMD_FS, ...`, could local caller call it straight instead of `ree_fs_rpc_xxx(...)`. It seems that there will not be anything 'ree fs' specific to add inside these routines.